xds/third_party/envoy/src/main/proto/envoy/api/v2/auth/cert.proto

```diff
@@ -13,9 +13,9 @@ import "envoy/api/v2/core/config_source.proto";
 import "google/protobuf/wrappers.proto";
 
 import "validate/validate.proto";
-import "gogoproto/gogo.proto";
 
-option (gogoproto.equal_all) = true;
```

Comment:
This is wrong, I wonder how you did the import.

Reply:
Ohh, you changed the script in last month~~  I was using my "ancient" script that can run on macOS to bring in the proto files from envoy's repo and then change the commit number in HEAD's import.sh. LOL.

Thanks catching this!